### ISSUE TITLE
docker - "bin/sh: npm: not found"

### ISSUE THREAD

**Issue by the-herbivore** [mwielgoszewski/doorman#109]
Seeing this error when attempting a `docker build -t doorman:Latest .`

```
...
Executing busybox-1.26.2-r5.trigger
Executing ca-certificates-20161130-r2.trigger
OK: 293 MiB in 64 packages
Requirement already up-to-date: pip in /usr/lib/python2.7/site-packages
/bin/sh: npm: not found
The command '/bin/sh -c echo "http://dl-cdn.alpinelinux.org/alpine/edge/community" >> /etc/apk/repositories   && apk add --update               bash               build-base               git               libffi-dev               musl               nodejs               postgresql-dev               py2-pip               python               python-dev               redis               runit   && pip install --upgrade pip   && npm install -g bower less   && rm /var/cache/apk/*' returned a non-zero code: 127
```

```
$ which npm
/usr/bin/npm
```

Something I've done wrong? Any clues?

**Comment by dmoore44** [comment]
Edit the dockerfile to include the nodejs-npm package.

**Comment by mwielgoszewski** [comment]
Should be fixed in a3350c51c7223e366419a85ca8742a78871a2dab. Thanks to @bpetri.
